testutils/src/main/java/com/amplifyframework/testutils/ObjectValidatorUtils.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.testutils;
+
+import org.junit.Assert;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Utility class that can be used to test core
+ * Object behavior of a class that implements its own
+ * equals, hashCode and toString methods.
+ */
+public final class ObjectValidatorUtils {
```

Comment:
Okay, cool. This looks like a good general purpose testutil.

Gripes about naming (classic jhwill@):

- Let's drop the `Utils`
- From the method it contains, maybe `EqualsToStringHashValidator` is a reasonable name? It's kinda long, but it's very clear and narrow in its utility:

Reply:
Hahaha...I knew "Utils" was going to get some attention. I'm good with that and the method name as well. LIke I said before, I would have never made it as a marketing person.